src/jit/lower.h

```diff
@@ -246,15 +246,17 @@ class Lowering : public Phase
     // operands.
     //
     // Arguments:
-    //     tree  -  Gentree of a binary operation.
+    //     tree  -             Gentree of a binary operation.
+    //     isSafeToMarkOp1     True if it's safe to mark op1 as register optional
```

Comment:
Here (or in a Notes section below) you should mention that it is expected that the caller has already determined this by calling `IsSafeToContainMem`

Reply:
Done